### DISCUSSION TITLE
Error when doing npm run dev

### DISCUSSION THREAD

**Discussion by JonTron1987** [vercel/next.js#36172] [Help]
Greetings, I am creating a Next.js site for a portfolio. This is my first time using Next.js. I have used React.js, but not Next. When I am trying to run my generated site for the first time I get this error below:

![image](https://user-images.githubusercontent.com/12778501/163470903-02f919b6-5d2f-450a-886a-167cb34c1d3c.png)
 
I used the default template using npx command to install. Did it not install a dependency?

**Comment by icyJoseph** [comment]
Well, this happens when the webpack config is wrong, plain and simple.

I'd guess that the boilerplate you used is out of date, or not maintained, could you share a link to make sure we are on the same page?

**Reply by JonTron1987** [reply]
yes, one sec

**Reply by JonTron1987** [reply]
npx create-next-app nextjs-blog --use-npm --example "https://github.com/vercel/next-learn/tree/master/basics/learn-starter", it was like this, but instead you put npx create-next-app (name of your site) --template default:

https://nextjs.org/docs
